cartofacade/testhelpers.go

```diff
@@ -19,14 +19,13 @@ func GetTestConfig(cameraName, movementSensorName string) (CartoConfig, string,
 		ComponentReference: "component",
 		LidarConfig:        TwoD,
 		CloudStoryEnabled:  false,
-		EnableMapping:      false,
+		EnableMapping:      true,
 	}, dir, nil
 }
 
 // GetBadTestConfig gets a sample config for testing purposes that will cause a failure.
 func GetBadTestConfig() CartoConfig {
 	return CartoConfig{
-		Camera:      "rplidar",
```

Comment:
[note] removing as "data_dir" can't be used as a required param anymore, must be camera